app/src/main/java/com/sopt/kakaobank/presentation/history/HistoryFragment.kt

```diff
@@ -1,11 +1,113 @@
 package com.sopt.kakaobank.presentation.history
 
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
 import com.sopt.kakaobank.R
 import com.sopt.kakaobank.core.base.BindingFragment
+import com.sopt.kakaobank.core.util.fragment.statusBarColorOf
+import com.sopt.kakaobank.data.dto.response.History
 import com.sopt.kakaobank.databinding.FragmentHistoryBinding
+import java.text.DecimalFormat
 
 class HistoryFragment : BindingFragment<FragmentHistoryBinding>(R.layout.fragment_history) {
     override fun initView() {
-        // 로직 작성
+        statusBarColorOf(R.color.main) //status 색상 변경
+        setBackButton()
+        setHistoryList()
     }
+
+    // 뒤로 가기
+    private fun setBackButton() {
```

Comment:
```suggestion
    private fun initBackBtnClickListener() {
```
init[View]ClickListener : 클릭 리스너 설정
setupXXX() : ViewModel을 observe()할때 모아놓는 함수 이름

저는 요 컨벤션을 따르는 편!